src/Illuminate/Queue/Worker.php

```diff
@@ -200,7 +200,10 @@ protected function stopIfNecessary(WorkerOptions $options, $lastRestart)
         }
 
         if ($this->memoryExceeded($options->memory)) {
-            $this->stop(12);
+            gc_collect_cycles();
```

Comment:
We might want to check if this function is available to call on the system before calling it.

Reply:
This function may be disabled on peoples installs, so this patch release might totally break their app.

Reply:
I see. We will check and consider it.